Assets/SEE/DataModel/DG/IO/GraphReader.cs

```diff
@@ -242,6 +241,18 @@ private static Stream OpenFile(string filename)
         /// </summary>
         private readonly string basePath;
 
+        /// <summary>
+        /// The attribute name for the source region line length.
+        /// We need to use this attribute for our SourceRange attribute.
+        /// </summary>
+        public const string RegionLengthAttribute = "Source.Region_Length";
```

Comment:
Yes, they are only relevant for the import and the export, so I don't think `GraphElement` is the most fitting location. Maybe there's a different better place for them, since the GraphWriter has to access these names too?

Reply:
Would it make sense to add a common abstract superclass `GraphIO` for `GraphWriter` and `GraphReader`, where these declaration could reside (and hidden)? 

We have something similar in the metrics import/export. It would also make the connection between `GraphWriter` and `GraphReader` explicit.

Reply:
This sounds like a good idea. I've started implementing it, but a problem I'm having is that `GraphReader` inherits from `GXLParser`, which is in `SEE.Utils` (and thus probably shouldn't depend on `GraphIO`). The only usage of `GXLParser` is as a superclass of `GraphReader`, so we could just move it over to the other namespace.

Reply:
Agreed. We should move `GXLParser` to the namespace of `GraphReader`.